app/editor/src/features/admin/event-of-the-day/hooks/useColumns.tsx

```diff
@@ -86,20 +87,38 @@ export const useColumns = (
     }
   };
 
+  const handleCopyTitle = (event: any, cell: any) => {
```

Comment:
Should this be a `useCallback` to stop re-renders?

Reply:
I'm not sure it would trigger a re-render as there's no change in state from anything happening in the method.  If I changed it to a `useCallback` it would have no dependencies.  That's the determination isnt it?

Reply:
I don't think that's entirely correct.  I think a function will result in a new function each time it is processed.  However, there is a trade off of placing everything into memory.